### PR TITLE
Update Swift SDK meeting notes URL

### PR DESCRIPTION
**Pull request by nachoBonafonte**

### CODE REVIEW

File: README.md
Comment (arminru) [inline README.md:143]: This document is not publicly accessible. Please make sure it's world-readable (maybe allow comment access, most SIGs even have them publicly editable although that's probably not ideal).

Please also make sure it's owned by the OTel GC account - see https://github.com/open-telemetry/community/issues/1147.

Comment (dyladan) [reply README.md:143]: @nachoBonafonte any update? The document is still private.

Comment (trask) [reply README.md:143]: ping @open-telemetry/swift-maintainers

**Comment by nachoBonafonte** [comment]
Sorry for the delay, I totally forgot about it. I have tried assigning ownership as explained in #1147  but was not able to do it. Transfer ownership does not appear as an option after adding cncf as editor. At the end I have created a public accessible file with my personal ownership.

**Comment by arminru** [comment]
@nachoBonafonte Now that you have a doc owned from your personal account, could you try following the steps @dyladan laid out in https://github.com/open-telemetry/community/issues/1147#issuecomment-1234759420? I think now passing ownership should work.

**Comment by nachoBonafonte** [comment]
I created the ownership transfer now, waiting for the acceptance

**Comment by trask** [comment]
> I created the ownership transfer now, waiting for the acceptance

accepted, thanks!

**Review by trask — APPROVED** [review]

**Comment by trask** [comment]
@open-telemetry/governance-committee can we get one more approval on this? thx!

**Comment by trask** [comment]
@open-telemetry/governance-committee we still need another approval on this

@nachoBonafonte can you resolve the merge conflict? (sorry we didn't get this merged before conflicts arose)

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**
